it/uk/gov/hmrc/breathingspaceifproxy/support/WireMockSupport.scala

```diff
@@ -31,12 +27,15 @@ import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Suite}
 import play.api.http.{HeaderNames, Status}
 import play.mvc.Http.MimeTypes
 
+import java.net.URL
+import scala.jdk.CollectionConverters._
+
 trait WireMockSupport extends BeforeAndAfterAll with BeforeAndAfterEach {
   suite: Suite =>
 
   val wireMockPort = 12345
   val wireMockHost = "localhost"
-  val wireMockBaseUrl = new URL(s"http://$wireMockHost:$wireMockPort")
+  val wireMockBaseUrl = new URL(s"https://$wireMockHost:$wireMockPort")
```

Comment:
https on wiremock? Are you sure?

Reply:
Well, It does work. Wiremock can work on https. And the prime reason I fixed it is because it gives the warning for non secure url in Idea. I can change it back to http if needed.

Reply:
Wiremock server has not got a legitimate certificate so I would be surprised if it works. wireMockBaseUrl is not used at all. Can you delete it?